src/dialogs/more-info/controls/more-info-camera.ts

```diff
@@ -8,39 +8,41 @@ import {
   css,
 } from "lit-element";
 
-import computeStateName from "../../../common/entity/compute_state_name";
 import { HomeAssistant, CameraEntity } from "../../../types";
-import { fireEvent } from "../../../common/dom/fire_event";
 import {
-  fetchStreamUrl,
-  computeMJPEGStreamUrl,
   CAMERA_SUPPORT_STREAM,
   CameraPreferences,
   fetchCameraPrefs,
   updateCameraPrefs,
 } from "../../../data/camera";
 import { supportsFeature } from "../../../common/entity/supports-feature";
+import "../../../components/ha-camera-stream";
 import "@polymer/paper-checkbox/paper-checkbox";
 // Not duplicate import, it's for typing
 // tslint:disable-next-line
 import { PaperCheckboxElement } from "@polymer/paper-checkbox/paper-checkbox";
 
-type HLSModule = typeof import("hls.js");
-
 class MoreInfoCamera extends LitElement {
   @property() public hass?: HomeAssistant;
   @property() public stateObj?: CameraEntity;
   @property() private _cameraPrefs?: CameraPreferences;
-  private _hlsPolyfillInstance?: Hls;
 
   public disconnectedCallback() {
     super.disconnectedCallback();
-    this._teardownPlayback();
+    this.stateObj = undefined;
   }
 
   protected render(): TemplateResult | void {
+    if (!this.hass || !this.stateObj) {
+      return html``;
+    }
+
     return html`
-      <div id="root"></div>
+      <ha-camera-stream
+        .hass="${this.hass}"
+        .stateObj="${this.stateObj}"
+        showControls="true"
```

Comment:
`showcontrols` should be enough.

Reply:
Since attributes are case insensitive, `showcontrols` will map to setting `showControls` to `true` in lit.

Reply:
```suggestion
        showcontrols
```